tilt_right_controller:
  controller:
    package: dynamixel_controllers
    module: joint_position_controller
    type: JointPositionController
  joint_name: tilt_right_joint
  joint_speed: 1.0
  motor:
    id: 1
    init: 512
    min: 450
    max: 800

tilt_left_controller:
  controller:
    package: dynamixel_controllers
    module: joint_position_controller
    type: JointPositionController
  joint_name: tilt_left_joint
  joint_speed: 1.0
  motor:
    id: 2
    init: 512
    min: 300
    max: 650

pan_controller:
  controller:
    package: dynamixel_controllers
    module: joint_position_controller
    type: JointPositionController
  joint_name: pan_joint
  joint_speed: 1.5
  motor:
    id: 3
    init: 512
    min: 380
    max: 560
